helm/microgateway/requirements.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
@Savidude since, there are no dependencies, the `requirements.yaml` is not required.